prime-router/src/test/kotlin/fhirengine/engine/fhirRouterTests/RoutingTests.kt

```diff
@@ -102,17 +125,146 @@ class RoutingTests {
         clearAllMocks()
     }
 
+    @Test
+    fun `test replaceShorthand with simple string`() {
+        val settings = FileSettings().loadOrganizations(twoOrganization)
+        val engine = spyk(makeFhirEngine(metadata, settings, TaskAction.route) as FHIRRouter)
+
+        // act
+        val result = engine.replaceShorthand("%patient")
+
+        // assert
+        assertThat(result).isEqualTo("Bundle.entry.resource.ofType(Patient)")
+    }
+
+    @Test
+    fun `test replaceShorthand with two variable string`() {
+        val settings = FileSettings().loadOrganizations(twoOrganization)
+        val engine = spyk(makeFhirEngine(metadata, settings, TaskAction.route) as FHIRRouter)
+
+        // act
+        val result = engine.replaceShorthand("%patient or %messageId")
```

Comment:
You could just remove this and other similar vals and just put the code in the assertThat()